pod-operation/src/components/gyro.rs

```diff
@@ -0,0 +1,19 @@
+use mpu6050::Mpu6050;
+use rppal::i2c::I2c;
+
+pub struct Mpu6050Sensor {
+	mpu6050: Mpu6050<I2c>,
+}
+
+impl Mpu6050Sensor {
```

Comment:
Chore: while we are using the mpu6050 sensor internally, it would be more useful for a consumer to see how this component fits into the pod: could we rename this struct to `Gyroscope`